test/mocks.js

```diff
@@ -93,15 +102,13 @@ function mockDB (data, errors) {
     devices: sinon.spy(function () {
       return P.resolve([])
     }),
-    deleteVerificationReminder: sinon.spy(function () {
-      return P.resolve()
```

Comment:
These empty spies can be removed because the default method body provided by `mockObject` defines them like this automatically.